.scalafmt.conf

```diff
@@ -1,23 +1,45 @@
-version = 2.7.5
+version = 3.0.8
 
 style = default
 
 maxColumn = 100
+docstrings.wrap = false
 
 // Vertical alignment is pretty, but leads to bigger diffs
 align.preset = none
 
+# Commented-out configs are Good Ideas but will create a big initial diff if applied
+
 danglingParentheses.preset = false
+# danglingParentheses.preset = true
 
 rewrite.rules = [
   AvoidInfix
   RedundantBraces
   RedundantParens
-  AsciiSortImports
   PreferCurlyFors
+  # SortModifiers
+]
+
+rewrite.sortModifiers.order = [
+  override, implicit, private, protected, final, sealed, abstract, lazy
 ]
 
+# rewrite.trailingCommas.style = multiple
+
 project.excludeFilters = [
    "scalafix-inputs",
    "scalafix-outputs"
 ]
+
+runner.dialect = scala212
+
+fileOverride {
+  "glob:**/scala-3/**/*.scala" {
+    runner.dialect = scala3
+    # rewrite.scala3.preset = true
```

Comment:
I don't know what this does.

Reply:
It removes optional braces and uses new control flow syntax.

Reply:
But we can't use it because `-source:3.0-migration` is enabled.

Reply:
Oh.  Well, good.  I don't like that new syntax anyway!